posts/2022-11-14-using-codeengine-with-paketo-buildpack-for-liberty.adoc

```diff
@@ -8,8 +8,8 @@ author_github: https://github.com/kevin-ortega
 seo-title:  Deploy Java applications to IBM Cloud Code Engine (ICCE) using the Paketo Buildpack for Liberty - OpenLiberty.io
 seo-description: Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty
 blog_description: "Learn to deploy your applications to ICCE easily with the Paketo Buildpack for Liberty"
-open-graph-image: https://openliberty.io/img/blog/code_engine_build_step_details.png
-open-graph-image-alt: screen capture of Code Engine build details
+open-graph-image: /img/blog/code_engine_build_strategy.png
```

Comment:
```suggestion
open-graph-image: https://openliberty.io/img/blog/code_engine_build_step_details.png
```

Return it back to absolute URL based on my findings in comment https://github.com/OpenLiberty/blogs/pull/2748#discussion_r1021997342